src/SDKs/MachineLearningCompute/Management.MachineLearningCompute/Microsoft.Azure.Management.MachineLearningCompute.csproj

```diff
@@ -4,7 +4,7 @@
         <PackageId>Microsoft.Azure.Management.MachineLearningCompute</PackageId>
         <Description>Provides developers with a library to create and manage machine learning compute resources.</Description>
         <AssemblyName>Microsoft.Azure.Management.MachineLearningCompute</AssemblyName>
-        <Version>0.1.0</Version>
+        <Version>0.2.0</Version>
         <PackageTags>Microsoft Azure Machine Learning Compute management;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
         <PackageReleaseNotes>This is a preview release of the Machine Learning Compute SDK.</PackageReleaseNotes>
```

Comment:
@shutchings Update PackageReleaseNotes and provide release notes about your package.

Reply:
Good catch!